README.md

```diff
@@ -10,15 +10,17 @@ All the credit here goes to the incredible team at Mailbox for changing the way
 
 1. copy email_snooze.py to a directory where you want it to live (preferrably on a server somewhere, we rely on cron to run this every day)
 
-2. edit the users = line (~5th to last line in the file) to have your email addresses, or at least the right number of emails you'd like to use with this (the actual strings are all relative to issued OAuth tokens in the app, so only used for debugging)
+2. run "pip install -r requirements.txt" to install the needed dependencies
```

Comment:
GitHub makes this diff look super confusing. All I've done is added this line, and incremented the numbers in the following steps.